src/ucp/rndv/rndv_rtr.c

```diff
@@ -239,29 +238,26 @@ static void ucp_proto_rndv_rtr_query(const ucp_proto_query_params_t *params,
 }
 
 static void 
-ucp_proto_rndv_rtr_abort_super(void *req, ucs_status_t status, void *user_data)
+ucp_proto_rndv_rtr_abort_super(void *request, ucs_status_t status, void *user_data)
 {
-    ucp_request_t *ucp_req = (ucp_request_t*)req - 1;
-    ucp_proto_rndv_recv_complete(ucp_req);
+    ucp_request_t *req = (ucp_request_t*)request - 1;
+    ucp_proto_rndv_recv_complete(req);
 }
 
 static void ucp_proto_rndv_rtr_abort(ucp_request_t *req, ucs_status_t status)
 {
     const ucp_proto_rndv_rtr_priv_t *rpriv = req->send.proto_config->priv;
     ucp_request_t *rreq                    = ucp_request_get_super(req);
+    ucp_mem_h memh                         = req->send.state.dt_iter.type.contig.memh;
```

Comment:
are we sure the datatype is contig?

Reply:
Acquiring memh moved inside `memh_invalidate`.